src/components/RegistrationList.tsx

```diff
@@ -66,6 +88,28 @@ export const RegistrationListItemContent = ({
   const publicationDate = displayDate(entityDescription?.publicationDate);
   const heading = [typeString, publicationDate].filter(Boolean).join(' — ');
 
+  const isPromotedPublication = promotedPublications && promotedPublications.includes(identifier);
```

Comment:
har forstått det på Kir som at vi ønsker å persistere `id` og ikke `identifier`(?). Det er også mer i tråd med hvordan vi ellers persisterer ting
```suggestion
  const isPromotedPublication = promotedPublications && promotedPublications.includes(id);
```